documentation/user_docs/docs/user_guide/Getting_started.rst

```diff
@@ -16,7 +16,7 @@ Before starting you must first extract the data files from the supplied zip file
 Creating an SQW file
 ====================
 
-The first step in using Horace is to make your dataset from all of your relevant SPE files. How this works depends somewhat on the properties of your computer, specifically the amount of memory available, and is dealt with :ref:`here <Download_and_setup:System Requirements>`. On most machines (on which the available RAM is <10GB) the dataset is written to a new file with the extension .SQW, and intermediate .TMP files, which contain axes projection information, are written as Horace combines the data. Once you have created your SQW file and are happy with it then you can delete these intermediate .TMP files if you wish, although it is generally a good idea to keep for a few days them unless disk space is a problem for you, in case you wish to re-generate your SQW file. For special cases where large amounts of memory are available then the creation of .TMP files is unnecessary and the SQW file can be created directly. This latter case is dealt with in the section of this manual detailing :ref:`advanced use <Advanced_use:Advanced use>`, for the rest of the following we shall assume you are running a machine with less memory.
+The first step in using Horace is to make your dataset from all of your relevant SPE files. How this works depends somewhat on the properties of your computer, specifically the amount of memory available, and is dealt with :ref:`here <introduction/Download_and_setup:System Requirements>`. On most machines (on which the available RAM is <10GB) the dataset is written to a new file with the extension .SQW, and intermediate .TMP files, which contain axes projection information, are written as Horace combines the data. Once you have created your SQW file and are happy with it then you can delete these intermediate .TMP files if you wish, although it is generally a good idea to keep for a few days them unless disk space is a problem for you, in case you wish to re-generate your SQW file. For special cases where large amounts of memory are available then the creation of .TMP files is unnecessary and the SQW file can be created directly. This latter case is dealt with in the section of this manual detailing :ref:`advanced use <user_guide/Advanced_use:Advanced use>`, for the rest of the following we shall assume you are running a machine with less memory.
```

Comment:
Slight ambiguity. "on most machines (on which available RAM <10GB)" - do you mean most of the machines in the population of these with <10GB, or all machines with <10GB, which is probably the majority.